Update log-analytics-powerbi.md

ehissey opened the pull request:
PG owner for this page.

Adding 2 points to the note at the top of the page:
1) That the power bi export option is no longer in Preview settings for upgraded workspaces, as the feature is a GA feature in Search V2.
2) That users may run into access problems in the Power BI export step since this requires Azure RBAC. Many users may only have Classic Portal access to a workspace, but not have access to the subscription it is tied to, denying them access when Power BI desktop makes the ARM call using their creds to the workspace.

tysonn commented:
Jason, we need to use sentence casing in headings. Can you please update your committed changes to use sentence casing?

tysonn commented:
Second notice: Jason, we need to use sentence casing in headings. Can you please update your committed changes to use sentence casing?

ehissey committed:
Update log-analytics-powerbi.md

Adding 2 points to the note at the top of the page:
1) That the power bi export option is no longer in Preview settings for upgraded workspaces, as the feature is a GA feature in Search V2.
2) That users may run into access problems in the Power BI export step since this requires Azure RBAC. Many users may only have Classic Portal access to a workspace, but not have access to the subscription it is tied to, denying them access when Power BI desktop makes the ARM call using their creds to the workspace.
